workflow/resources/config/scenario.yaml

```diff
@@ -11,14 +11,20 @@ __definitions__:
     samples = params.samples.set_index("alias")
     if "ffpe" not in samples.columns:
       samples["ffpe"] = pd.NA
-  - sex = samples.loc["tumor", "sex"]
+  - sex = samples.loc[["tumor"], "sex"]
```

Comment:
In case of groups with just one entry `sample.loc["tumor", "sex"]` will just return sex as a string.
But if there are multiple entries for a group sex will become a series.
In the previous implementation rendering the scenario only worked for groups with a single entry.
Changing sex to `sample.loc[["tumor"], "sex"]` will always return a series allowing single and multiple entries correctly.